### ISSUE TITLE
Handle Errors

### ISSUE THREAD

**Issue by DakuTree** [DakuTree/manga-tracker#43]
So I just had my first case of everything going to hell because I had no idea what was causing the problem.
Basically I marked a bato.to chapter as read & for whatever reason it didn't parse the chapter number properly yet it was still successfully submitted, this caused the chapter parser to fail & caused an error, which in turn caused a output error. I also didn't have any error logs of what was even going on for some reason.

Basically, I need to stop trying to take the fast route of doing things.
